metadata.rb

```diff
@@ -4,21 +4,25 @@
 license          'Apache 2.0'
 description      'Installs Microsoft Visual C++ Runtimes'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.2.1'
+version          '0.2.14'
 
 %w( windows ).each do |os|
   supports os
 end
 
-depends 'windows'
-
+provides 'vcruntime::vc6'
```

Comment:
i thought it was for the supermarket to know certain stuff, it was in the original cookbook so i left it and updated it. I'm still kinda fresh to chef.